tests/functional/firefox/os/version/test_base.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.os.version.base import FirefoxOSBasePage
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('version', [('2.5'), ('2.0'), ('1.4'), ('1.3t'), ('1.3'), ('1.1')])
+def test_family_navigation(version, base_url, selenium):
+    page = FirefoxOSBasePage(base_url, selenium, version=version).open()
+    page.family_navigation.open_menu()
+    assert page.family_navigation.is_menu_displayed
+
+
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('version', [('2.0'), ('1.4'), ('1.3t'), ('1.3'), ('1.1')])
+def test_news_is_displayed(version, base_url, selenium):
+    page = FirefoxOSBasePage(base_url, selenium, version=version).open()
+    assert page.is_news_displayed
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('version', [('2.0'), ('1.4'), ('1.3t'), ('1.3'), ('1.1')])
+def test_cta_buttons_are_present(version, base_url, selenium):
+    page = FirefoxOSBasePage(base_url, selenium, version=version).open()
+    # test for presence of buttons only as display is dependent upon geo location
```

Comment:
In Firefox it should be possible to fake the geolocation data by setting **geo.prompt.testing** to **true**, **geo.prompt.testing.allow** to **true**, and **geo.wifi.uri** to a data URL with JSON content describing the fake location. I've not tested this recently, and it would require some work, but I wanted to let you know that it should be possible, if this was an important feature to test.